docs/content/install.md

```diff
@@ -31,7 +31,8 @@ curl https://cdn.porter.sh/latest/install-linux.sh | bash
 ```
 
 ## Latest Windows
-You will need to create a [PowerShell Profile][ps-link] if you do not have one.
+
+You need to run this command from PowerShell. You will also need a [PowerShell Profile][ps-link] if you do not have one. The PowerShell Profile customizes your environment each time you start up PowerShell. 
 
 ```
 iwr "https://cdn.porter.sh/latest/install-windows.ps1" -UseBasicParsing | iex
```

Comment:
@gaurimadhok Did you say that `iwr` wasn't a valid command in your powershell?

Reply:
valid in powershell, not in a regular command prompt.

Reply:
Oops, I get it know, thanks!